src/main/java/model/LottoGenerator.java

```diff
@@ -0,0 +1,6 @@
+package model;
+
+public interface LottoGenerator {
```

Comment:
이와 같이 ``추상 메서드가 단 하나 뿐인 인터페이스``를 함수형 인터페이스라고 부릅니다. 
그리고 이런 함수형 인터페이스는 `@FunctionalInterface` 애너테이션을 붙입니다. (이 애너테이션의 역할에 대해서는 추상메서드를 하나 더 추가해보면 금방 알 수 있을 것 같아요~ 😉) 

```suggestion
@FunctionalInterface
public interface LottoGenerator {
```

Reply:
놓쳤던 부분인 것 같습니다. 감사합니다.